update_service/installing-update-service.adoc

```diff
@@ -0,0 +1,58 @@
+[id="installing-update-service"]
+= Installing and configuring the OpenShift Update Service
+include::modules/common-attributes.adoc[]
+:context: update-service
+
+toc::[]
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+You can install and configure the OpenShift Update Service to provide over-the-air updates for your cluster and its underlying operating system.
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+[id="update-service-prereqs"]
+== Prerequisites
+
+* For more information on installing Operators, see xref:../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
+
+[id="update-service-install"]
+== Installing the OpenShift Update Service Operator
+
+The OpenShift Update Service is installed by first installing the OpenShift Update Service Operator by using the {product-title} web console or CLI.
+
+include::modules/update-service-install-web-console.adoc[leveloffset=+2]
+
+include::modules/update-service-install-cli.adoc[leveloffset=+2]
+
+include::modules/update-service-graph-data.adoc[leveloffset=+2]
+
+[id="update-service-create-service"]
+== Creating an OpenShift Update Service application
+
+You can create an OpenShift Update Service application by using the {product-title} web console or CLI.
+
```

Comment:
For the clusters that are installed in a restricted network, OLM by default cannot access the OperatorHub sources. So the osus operator is not able to be installed directly if OLM is not customized in restricted network. I would suggest to have a note to explain it. For more information, see [Using Operator Lifecycle Manager on restricted networks](https://docs.openshift.com/container-platform/4.6/operators/admin/olm-restricted-networks.html).